gradle.properties

```diff
@@ -48,20 +48,20 @@ spark_40_antlr_version=4.13.1
 spark_33_jackson_version=2.13.4
 spark_34_jackson_version=2.14.2
 spark_35_jackson_version=2.15.2
-spark_40_jackson_version=2.17.0
+spark_40_jackson_version=2.18.2
 
 spark_33_slf4j_version=1.7.32
 spark_34_slf4j_version=2.0.6
 spark_35_slf4j_version=2.0.7
-spark_40_slf4j_version=2.0.7
+spark_40_slf4j_version=2.0.16
 
 # Align with Apache Spark, and don't bundle them in release jar.
 commons_lang3_version=3.12.0
```

Comment:
3.17.0, otherwise some enum, e.g., `JAVA_24`, which does not exist in older versions, may cause runtime linkage issues